users/tests/test_create_teams.py

```diff
@@ -16,9 +16,10 @@ def test_create_team_without_title(self):
 
     @pytest.mark.django_db(transaction=True)
     def test_create_team_with_same_name(self):
-        assert len(Team.objects.all()) == 0
+        length = len(Team.objects.all())
+        assert len(Team.objects.all()) == length
```

Comment:
This assert looks True forever it is the same as:
        assert len(Team.objects.all()) == len(Team.objects.all())
I think you can get rid of that in this case